docs/pages/management/guides/elastic-stack.mdx

```diff
@@ -0,0 +1,729 @@
+
+You will refer to this file later when configuring the plugin.
+
+</TabItem>
+<TabItem label="Connect to the Auth Service">
+
+If your network allows your plugin to access the Auth Service directly, e.g.,
+you are running the plugin on the Auth Service host, the plugin uses TLS
+credentials to connect to the Auth Service's gRPC endpoint and listen for audit
+events.
+
+You can generate TLS credentials with the following command:
+
+```code
+$ tctl auth sign --format=tls --user=teleport-event-handler --out=auth
+```
+
+This command should result in three PEM-encoded files: `auth.crt`,
+`auth.key`, and `auth.cas` (certificate, private key, and CA certs
+respectively). Later, you will configure the plugin to use these credentials to
+connect to the Auth Service.
+
+</TabItem>
+</Tabs>
+
+</ScopedBlock>
+
+<ScopedBlock scope="cloud">
+
+The following `tctl auth sign` command impersonates the `teleport-event-handler`
+user, generates signed credentials, and writes an identity file to the local
+directory:
+
+```code
+$ tctl auth sign --user=teleport-event-handler --out=auth.pem
+```
+
+Teleport's Event Handler plugin listens for new and updated audit events by
+connecting to the Teleport Auth Service's gRPC endpoint over TLS.
+
+The identity file, `auth.pem`, includes both TLS and SSH credentials. The Event
+Handler plugin uses the SSH credentials to connect to the Proxy Service, which
+establishes a reverse tunnel connection to the Auth Service. The plugin uses
+this reverse tunnel, along with your TLS credentials, to connect to the Auth
+Service's gRPC endpoint.
+
+You will refer to this file later when configuring the plugin.
+
+</ScopedBlock>
+
+<Admonition
+  title="Certificate Lifetime"
+>
+
+  By default, `tctl auth sign` produces certificates with a relatively short
+  lifetime. For production deployments, you can use the `--ttl` flag to ensure a
+  more practical certificate lifetime, e.g., `--ttl=8760h` to export a one-year
+  certificate.
+
+</Admonition>
+
+## Step 2/4. Configure a Logstash pipeline
+
+The Event Handler plugin forwards audit logs from Teleport by sending HTTP
+requests to a user-configured endpoint. We will define a Logstash pipeline that
+handles these requests, extracts logs, and sends them to Elasticsearch.
+
+### Create a role for the Event Handler plugin
+
+Your Logstash pipeline will require permissions to create and manage
+Elasticsearch indexes and index lifecycle management policies, plus get
+information about your Elasticsearch deployment. Create a role with these
+permissions so you can later assign it to the Elasticsearch user you will
+create for the Event Handler.
+
+In Kibana, navigate to "Management" > "Roles" and click "Create role". Enter the
+name `teleport-plugin` for the new role. Under the "Elasticsearch" section,
+under "Cluster privileges", enter `manage_index_templates`, `manage_ilm`, and
+`monitor`.
+
+Under "Index privileges", define an entry with `audit-events*` in the "Indices"
```

Comment:
Not sure if this matters or not, but the pattern specified in `audit-events.json` includes a `-` after `events`.